src/metrics/decorators/common.ts

```diff
@@ -1,5 +1,9 @@
 import { Counter, MetricOptions } from '@opentelemetry/api-metrics';
-import { getOrCreateCounter, getOrCreateValueRecorder, MetricType } from '../metric-data';
+import { getOrCreateCounter, MetricType } from '../metric-data';
+
+interface WhatIsThisShit {
```

Comment:
😂

Reply:
lmao, and thats what happens when you get frustrated about your IDEs code formatter. 😅
My bad dude, its gone now.